examples/intensity-recipe.yaml

```diff
@@ -83,25 +83,23 @@ Intensity:
                     Name: J/psi
                     Helicity: -1
                   DecayProducts:
-                    Particle:
-                      - Name: f2(1270)
-                        FinalState: [3, 4]
-                        Helicity: 0.0
-                      - Name: gamma
-                        FinalState: 2
-                        Helicity: -1
+                    - Name: f2(1270)
+                      FinalState: [3, 4]
+                      Helicity: 0.0
+                    - Name: gamma
+                      FinalState: 2
+                      Helicity: -1
```

Comment:
Is the expertsystem outputing this?

Reply:
Yeah this was requested in https://github.com/ComPWA/expertsystem/issues/73